src/app/state/staking/saga.ts

```diff
@@ -190,6 +190,10 @@ export function* getValidatorDetails({ payload: address }: PayloadAction<string>
 }
 
 export function* fetchAccount({ payload: address }: PayloadAction<string>) {
+  if (!isValidAddress(address)) {
+    return
```

Comment:
I'd remove if we don't need

Reply:
We don't strictly need it at this point, so I removed